.gitignore

```diff
@@ -8,8 +8,4 @@
 *.iml
 
 # bin
-/target/
-
-# xjc generation
-src/main/java/org/lsc/configuration/
-lsc.episode
\ No newline at end of file
+/target/
```

Comment:
nope, having multiple commits is fine. That's just your first and your third commit that should be squashed together. You can do this easily with git rebase -i